apps/web/src/config/constants/zapV3.ts

```diff
@@ -1,22 +1,56 @@
-export const ZAP_V3_POOL_ADDRESSES = [
-  '0x36696169C63e42cd08ce11f5deeBbCeBae652050', // USDT-WBNB 0.05%
-  '0x4f31Fa980a675570939B737Ebdde0471a4Be40Eb', // USDT-USDC 0.05%
-  '0x92b7807bF19b7DDdf89b706143896d05228f3121', // USDT-USDC 0.01%
-  '0xD4dCA84E1808da3354924cD243c66828cf775470', // ETH-BTCB 0.25%
-  '0x46Cf1cF8c69595804ba91dFdd8d6b960c9B0a7C4', // USDT-BTCB 0.05%
-  '0x133B3D95bAD5405d14d53473671200e9342896BF', // Cake-WBNB 0.25%
-  '0x6bbc40579ad1BBD243895cA0ACB086BB6300d636', // BTCB-WBNB 0.05%
-  '0x4BBA1018b967e59220b22Ca03f68821A3276c9a6', // ETH-BTCB 0.05%
-  '0x7f51c8AaA6B0599aBd16674e2b17FEc7a9f674A1', // Cake-USDT 0.25%
-  '0x172fcD41E0913e95784454622d1c3724f546f849', // USDT-WBNB 0.01%
-  '0xD0e226f674bBf064f54aB47F42473fF80DB98CBA', // ETH-WBNB 0.05%
-  '0xfab21Cb9467e9BaDd22A2dE57BCDE5F53D925973', // USDT-BNX 0.05%
-  '0x4f3126d5DE26413AbDCF6948943FB9D0847d9818', // USDT-BUSD 0.01%
-  '0x7d05c84581f0C41AD80ddf677A510360bae09a5A', // ETH-WBNB 0.25%
-  '0xFC75f4E78bf71eD5066dB9ca771D4CcB7C1264E0', // BTCB-WBNB 0.25%
-  '0x1E4600929Edf7F36B4A7eAc4C7571057D82a246c', // SOL-WBNB 0.25%
-  '0xBe141893E4c6AD9272e8C04BAB7E6a10604501a5', // ETH-USDT 0.05%
-  '0xd881D9d0e0767719701305c614903F555d589586', // TUSD-USDT 0.01%
-  '0xf2688Fb5B81049DFB7703aDa5e770543770612C4', // USDC-WBNB 0.01%
-  '0xd15B00E81F98A7DB25f1dC1BA6E983a4316c4CaC', // XRP-WBNB 0.25%
-]
+import { ChainId } from '@pancakeswap/chains'
+
+export const ZAP_V3_POOL_ADDRESSES: Record<number, string[]> = {
+  [ChainId.BSC]: [] as string[], // Empty array to whitelist all pools
```

Comment:
It's a bit counter intuitive. Can we use undefined check instead? Like if not specified then no whitelist restriction

Reply:
ah can you add a comment here? It would be super hard for other devs to understand without additional comment.

Or it would be better if we can have a separate constant SUPPORTED_CHAINS